lib/src/signed_video_h26x_sign.c

```diff
@@ -518,7 +518,7 @@ signed_video_add_nalu_part_for_signing_with_timestamp(signed_video_t *self,
 
     // Finalize GOP hash and generate SEI if the NALU is I frame of a non-empty GOP.
     if (nalu.is_first_nalu_in_gop && nalu.is_last_nalu_part) {
-      if (self->generate_sei) {
+      if (self->first_signing_attempt) {
```

Comment:
The name of the flag indicates that this can only happen once. "if this is first signing attempt then do...", but right now it enters the if statement also on the second, third, fourth, ... attempts